faucet.go

```diff
@@ -137,6 +137,13 @@ func verifyTimeLimit(ip string, actionsTimeLimit time.Duration) error {
 // getRealIP returns the clients IP address. If the useRealIP config is set
 // it will use the X-Real-IP or X-Forwarded-For HTTP headers, otherwise it will
 // return the request.RemoteAddr field.
+//
+// You should only use this option if you can trust the headers passed to
+// you (in particular, X-Forward-For and X-Real-IP) by a proxy.
+// If your reverse proxies are configured to pass along arbitrary header
+// values from the client, or if you use this option without a reverse
+// proxy, malicious clients will be able to make you very sad (or, depending on
+// how you're using RemoteAddr, vulnerable to an attack of some sort).
```

Comment:
I know this is just a comment, but copy/paste should still get a reference to the source (https://github.com/go-chi/chi/blob/0e733fe0c94055a67fdbc0534b63d6ad4e905320/middleware/realip.go#L22-L28), or be rephrased.